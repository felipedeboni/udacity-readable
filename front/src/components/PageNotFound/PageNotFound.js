import React from 'react';
import './PageNotFound.css';

const PageNotFound = () => {
  return (
    <div className="page-not-found">
      <span className="errorCode">404</span>
      <span className="message">Page Not Found</span>
    </div>
  );
};

export default PageNotFound;
