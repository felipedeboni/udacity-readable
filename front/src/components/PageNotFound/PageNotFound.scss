.page-not-found {
  color: #BABABA;
  height: calc(100vh - 163px);
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;

  .errorCode {
    display: block;
    font-size: 40vw;
    line-height: 1;
    text-align: center;

    @media (min-width: 990px) {
      font-size: 396px
    }
  }

  .message {
    font-size: 10vw;
    line-height: 1;

    @media (min-width: 990px) {
      font-size: 99px
    }
   }
}
