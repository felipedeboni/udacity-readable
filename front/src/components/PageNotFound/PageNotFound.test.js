import React from 'react';
import { shallow } from 'enzyme';
import PageNotFound from './PageNotFound';

describe('Page not found', () => {
  it('renders without crashing', () => {
    const test = shallow(<PageNotFound />);

    expect(test).toMatchSnapshot();
  });
});
