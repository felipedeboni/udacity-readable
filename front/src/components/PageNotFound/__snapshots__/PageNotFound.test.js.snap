// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page not found renders without crashing 1`] = `
<div
  className="page-not-found"
>
  <span
    className="errorCode"
  >
    404
  </span>
  <span
    className="message"
  >
    Page Not Found
  </span>
</div>
`;
